core/src/epicli/data/common/ansible/playbooks/roles/kibana/templates/kibana.yml.j2

```diff
@@ -14,12 +14,25 @@
 # permissions and limitations under the License.
 
 # Description:
-# Default Kibana configuration for Open Distro.
+# https://www.elastic.co/guide/en/kibana/current/settings.html
+# Default Kibana configuration
+
 server.host: "{{ ansible_default_ipv4.address | default(ansible_all_ipv4_addresses[0]) }}"
-elasticsearch.hosts: 
-{% for host in groups['logging'] %}
+elasticsearch.hosts:
```

Comment:
I'm not so sure if this should or even can be done. I've tried deploying both "logging" and "openistro_for_elasticsearch" elastisearch clusters and then I get errors in kibana:
```
Jul 23 14:35:01 x1b1 kibana[23006]:  FATAL  [index_not_found_exception] no such index [.kibana], with { resource.type="index_or_alias" & resource.id=".kibana" & index_uuid="_na_" & index=".kibana" } :: {"path":"/.kibana/_count","query":{},"body":"{\"query\":{\"bool\":{\"should\":[{\"bool\":{\"must\":[{\"exists\":{\"field\":\"index-pattern\"}},{\"bool\":{\"must_not\":{\"term\":{\"migrationVersion.index-pattern\":\"6.5.0\"}}}}]}},{\"bool\":{\"must\":[{\"exists\":{\"field\":\"visualization\"}},{\"bool\":{\"must_not\":{\"term\":{\"migrationVersion.visualization\":\"7.3.1\"}}}}]}},{\"bool\":{\"must\":[{\"exists\":{\"field\":\"dashboard\"}},{\"bool\":{\"must_not\":{\"term\":{\"migrationVersion.dashboard\":\"7.3.0\"}}}}]}},{\"bool\":{\"must\":[{\"exists\":{\"field\":\"search\"}},{\"bool\":{\"must_not\":{\"term\":{\"migrationVersion.search\":\"7.0.0\"}}}}]}}]}}}","statusCode":404,"response":"{\"error\":{\"root_cause\":[{\"type\":\"index_not_found_exception\",\"reason\":\"no such index [.kibana]\",\"resource.type\":\"index_or_alias\",\"resource.id\":\".kibana\",\"index_uuid\":\"_na_\",\"index\":\".kibana\"}],\"type\":\"index_not_found_exception\",\"reason\":\"no such index [.kibana]\",\"resource.type\":\"index_or_alias\",\"resource.id\":\".kibana\",\"index_uuid\":\"_na_\",\"index\":\".kibana\"},\"status\":404}"}
Jul 23 14:35:01 x1b1 systemd[1]: kibana.service: Main process exited, code=exited, status=1/FAILURE
Jul 23 14:35:01 x1b1 systemd[1]: kibana.service: Failed with result 'exit-code'.
Jul 23 14:35:01 x1b1 systemd[1]: kibana.service: Service hold-off time over, scheduling restart.
Jul 23 14:35:01 x1b1 systemd[1]: kibana.service: Scheduled restart job, restart counter is at 96.
Jul 23 14:35:01 x1b1 systemd[1]: Stopped Kibana.
Jul 23 14:35:01 x1b1 systemd[1]: Started Kibana.
```
I'm afraid we can't just connect kibana to two different elasticsearch clusters :(. It seems kibana keeps its config in the `.kibana` index, it's created in one cluster, then kibana tries other nodes and sees there is no `.kibana` index. I think we have to pick just one! Please reconsider this solution.